content/blog/blog-2020-08-26-kudo-0.16.0.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2020-08-27
+---
+
+# Announcing KUDO 0.16.0
+
+We are happy to release [KUDO v0.16.0](https://github.com/kudobuilder/kudo/releases/tag/v0.16.0), with focus on upgradability, support for immutable parameters and a lot of other small improvements and bugfixes.
+
+<!-- more -->
+
+To see the full changelog and the list of contributors who contributed to this release, visit [the Github Release](https://github.com/kudobuilder/kudo/releases/tag/v0.16.0) page.
+
+## KUDO Upgrades
+
+Until recently, `kubectl kudo init` would simply apply the expected state to the cluster and ignore any breaking changes that may have been introduced. This is now changed, from now on:
+
+- `kubectl kudo init` will fail if it detects an existing KUDO installation to prevent accidental upgrades.
+- You can use `kubectl kudo init --upgrade` to upgrade an existing installation.
+
+The upgrade process will verify that the upgrade can be executed correctly and - in later versions - will check installed operators and may contain migrations and other features that have to be done in an upgrade. 
+
+In the same vein we started to integrate upgrade-tests to our CI to ensure that new KUDO releases won't break existing installations and can be upgraded with a defined process.
+
+Run `kubectl kudo init --help` for more details.
+
+## Immutable Parameters
```

Comment:
we should probably add details for viewing what is immutable with the `k kudo package list params`

Reply:
I've added a `::: tip` note, good idea.